### DISCUSSION TITLE
Merry Christmas and happy holidays!  Cross-posting from Reddit.

### DISCUSSION THREAD

**Discussion by bitbound** [immense/Remotely#198] [Show and tell]
Figured I'd share this here too.  😄 

https://www.reddit.com/r/remotely_app/comments/kjz1rd/merry_christmas_and_happy_holidays_here_are_some/

**Comment by swilkey** [comment]
Merry Christmas Jared (and everyone else), and thanks!

**Reply by pacoos** [reply]
Merry Christmas!!! excelent develop!!!! ;)
